_data/simple-icons.json

```
@@ -8918,7 +8919,7 @@
         },
         {
             "title": "OpenJDK",
-            "hex": "FFFFFF",
+            "hex": "000000",
```

```suggestion
            "hex": "437291",
```

Just this, and then I think we're good to go on this one!